metrics/group_data_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 Tigris Data, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"testing"
+)
+
+func TestNewDataMetricGroup(t *testing.T) {
+	metrics := getMetricsFromTestFile(t, "status-single-basic.json")
```

Comment:
Is this file supposed to be checked in?

Reply:
Was the file added prior?

Reply:
Yes, it was added prior, other tests are using it.